sound-effect-manager.js

```diff
@@ -133,16 +139,39 @@ SoundEffectManager.prototype._playWebAudio = function (soundName, loop) {
         return;
     }
 
-    var source = this.context.createBufferSource();
-    source.buffer = buffer;
-    source.loop = loop;
-    source.connect(this.context.destination);
-
-    if (loop) {
-        this.sources[soundName] = source;
+    function base64ToArrayBuffer(base64) {
+        var binaryString = window.atob(base64);
```

Comment:
This won't work in IE9 - https://developer.mozilla.org/en-US/docs/Web/API/WindowBase64/atob. Given that it worked previously in IE9 via XMLHttpRequest, maybe the solution is to always load through XMLHttpRequest is `atob` isn't present?